.drone.yml

```
@@ -1299,7 +1303,7 @@ steps:
     image: plugins/manifest
     pull: always
     settings:
-      auto_tag: true
+      auto_tag: false
```

```suggestion
      # Enable on the latest release branch only and tags
      # auto_tag: true
      tags: ${DRONE_TAG}
```
this time we dont trim as we have https://github.com/go-gitea/gitea/blob/main/docker/manifest.tmpl#L11